### PR TITLE
NIFILIBS-1 Setup root pom, README, LICENSE, & NOTICE

### PR DESCRIPTION
**Pull request by bbende**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+
+## Table of Contents
+
+- [Getting Started](#getting-started)
+- [License](#license)
+
+## Getting Started
+
+### Requirements
+
```
Comment (ottobackwards) [inline README.md:27]: Should this be Java 1.8 and newer now?

File: README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+### Requirements
+    
+* Java 1.8
+* Maven 3.1.0, or newer
+* Recent git client
+
+### Building
+
+1) Clone the repo
+
```
Comment (ottobackwards) [inline README.md:36]: Shouldn't this be:
```
     git clone https://git-wip-us.apache.org/repos/asf/nifi-standard-libraries.git
     cd nifi-standard-libraries
     git checkout master
```

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,521 @@
...
+        </mailingList>
+        <mailingList>
+            <name>Commits</name>
+            <subscribe>commits-subscribe@nifi.apache.org</subscribe>
+            <unsubscribe>commits-unsubscribe@nifi.apache.org</unsubscribe>
+            <post>commits@nifi.apache.org</post>
+            <archive>https://mail-archives.apache.org/mod_mbox/nifi-commits</archive>
+        </mailingList>
+    </mailingLists>
+
```
Comment (ottobackwards) [inline pom.xml:76]: These should be gitbox now I think

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,521 @@
...
+                                    <version>${maven.min-version}</version>
+                                </requireMavenVersion>
+                            </rules>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
```
Comment (ottobackwards) [inline pom.xml:311]: It would be nice to have this as a separate file in the project.

**Review by ottobackwards — COMMENTED** [review]
Just a couple of things that I can see

File: README.md (hunk 1 above)
Comment (kevdoran) [reply README.md:27]: I think for a while these libraries will have to be built using JDK 8 and should run on newer versions. We should probably add some information here:

JDK 1.8 for building.
JRE >= 1.8 for running or consuming these libraries.

File: pom.xml (hunk 4 above)
Comment (kevdoran) [reply pom.xml:311]: Yeah that's a good point. We could move this to a checkstyle.xml file at the root of the repo.

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,521 @@
...
+  limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache</groupId>
+        <artifactId>apache</artifactId>
+        <version>19</version>
```
Comment (kevdoran) [inline pom.xml]: Any reason not to use the latest version of the parent pom (21)?

**Review by kevdoran — COMMENTED** [review]
Thanks @bbende!

Overall looks good. I had one or two minor things in addition to what Otto caught, but aside from that all-in-all I think this is a pretty good starting point for the repo.

File: README.md (hunk 1 above)
Comment (bbende) [reply README.md:27]: Will update

File: README.md (hunk 2 above)
Comment (bbende) [reply README.md:36]: Will update

File: pom.xml (hunk 3 above)
Comment (bbende) [reply pom.xml:76]: I think they are all valid now, but I'll switch to gitbox

File: pom.xml (hunk 4 above)
Comment (bbende) [reply pom.xml:311]: Will update

File: pom.xml (hunk 5 above)
Comment (bbende) [reply pom.xml]: No reason, just took 19 b/c that is what NiFi had, will update

**Comment by bbende** [comment]
@kevdoran @ottobackwards thanks for taking a look, pushed another commit

**Comment by ottobackwards** [comment]
+1, thanks @bbende!

**Review by kevdoran — APPROVED** [review]
+1, will merge. Thanks @bbende!

**Pull request merged**